examples/node-false/README.md

```diff
@@ -0,0 +1,9 @@
+# CLI - node false
```

Comment:
these are more for "how to" for users, but there's no harm in adding this. the tests that matter are in `/test`. no worries if you can't get to adding one there, we can do it after merging.